From 4a909a03b6dd336e7ea76e3f44d7cfb5d7e44798 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Wed, 29 Nov 2023 12:49:35 +0530
Subject: [PATCH 2/3] cmake: Fix overloaded-virtual error

* Remove -Werror for:
|git/src/plugins/intel_gpu/src/kernel_selector/jitter.h:129:28: error: 'virtual kernel_selector::JitDefinitions kernel_selector::JitConstant::GetDefinitions() const' was hidden [-Werror=overloaded-virtual=]
|  129 |     virtual JitDefinitions GetDefinitions() const = 0;
|      |

Upstream-Status: Pending

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 src/plugins/intel_gpu/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/intel_gpu/CMakeLists.txt b/src/plugins/intel_gpu/CMakeLists.txt
index 2f3d9127dde..2fd4f5c1b3c 100644
--- a/src/plugins/intel_gpu/CMakeLists.txt
+++ b/src/plugins/intel_gpu/CMakeLists.txt
@@ -47,7 +47,7 @@ add_subdirectory(thirdparty)
 include(thirdparty/cmake/rapidjson.cmake)
 
 if(CMAKE_COMPILER_IS_GNUCXX)
-    ov_add_compiler_flags(-Werror)
+	#ov_add_compiler_flags(-Werror)
 endif()
 
 add_subdirectory(src/runtime)
-- 
2.34.1

